utility/mm2_crawler

```diff
@@ -1031,9 +1030,13 @@ def per_host(session, host, options, config):
     return rc
 
 
-def worker(options, config, host):
+def worker(options, config, host_id):
     global current_host
     global threads_active
+
+    session = mirrormanager2.lib.create_session(config['DB_URL'])
```

Comment:
The session has always been closed. The close() is further down and old code and does therefore not appear in the diff. close() is called in any case commit() only if no undefined behaviour has happened.